src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,18 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
     public static void main(String[] args) {
+        final Logger logger = LogManager.getLogger(Main.class);
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.info("Can not authenticate user. User - {}", "bob", e);
```

Comment:
```suggestion
            logger.error("Can not authenticate user. User - {}", "bob", e);
```

lets use error level here